### ISSUE TITLE
Extra params for test

### ISSUE THREAD

**Issue by Mathemalsky** [MilchRatchet/PAL#9]
I think it would be useful if running `TEST` had some optional arguments which can be used to suppress some categories of test. That would be very nice in the case of debugging. For example than you can simple test without the time consuming tests for convex hull. And also the valgrind outputs would be more clean than.

**Comment by MilchRatchet** [comment]
Yes, that sounds great. If you have an idea how we should do this then you can implement it.

**Comment by Mathemalsky** [comment]
My idea is to have several sets of test:
- `full` is the default set and tests everything
- `aal` tests just the algorithms from the AAL
- `cgl` tests just the algorithms from the CGL an the algorithms from VLL they depend on
- an argument for every single algorithm to test only this one

I think I can start implementing that soon.

**Comment by MilchRatchet** [comment]
Maybe we could define some 3 character abbreviation for each algorithm that can be used as an argument. Then we could call
```
./TEST CHJ CHC SCG
```
for testing CONVEXHULL_JARVIS, CONVEXHULL_CHAN and SETCOVER_GREEDY.

**Comment by Mathemalsky** [comment]
Also an idea. Maybe we can also do both.

**Comment by MilchRatchet** [comment]
Yes, that idea was meant for "an argument for every single algorithm to test only this one".
